clean.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+REM Copyright (c) .NET Foundation and contributors. All rights reserved.
+REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if "%~1"=="" (
+  @call run.cmd clean -?
+) else (

```

Comment:
How so? This check is for the number of arguments, not the result of `run.exe`...or am I reading the wrong line? :)